Block/Adminhtml/System/Config/MerchantInfoConfirm.php

```diff
@@ -0,0 +1,104 @@
+<?php
```

Comment:
With new PR we bump the version number, aren't we?

Reply:
It depends if you want to include more items on the release or consider this change a new version on it's own.
Technically it's a choice and personally I prefer to separate the version bumping from the actual change (most of the times).
We'll discuss that internally :)